source/reference/options.txt

```diff
@@ -76,6 +76,34 @@ General Options
 
    Returns the |mdb-shell| release number.
 
+.. _mongosh-stable-api-options:
+
+Stable API Options
+------------------
+
+.. option:: --apiVersion <version number>
+
+   Specifies the :ref:`apiVersion <api-version-desc>`. ``"1"`` is 
+   currently the only supported value.
+
+.. option:: --apiStrict
+
+   Specifies that the server will respond with :ref:`APIStrictError 
+   <api-strict-resp>` if your application uses a command or behavior 
+   outside of the :ref:`Stable API <stable-api>`.
+   
+   If you specify :option:`--apiStrict`, you must also specify
+   :option:`--apiVersion`.
```

Comment:
(question) Is this for apiStrict? From the description, it seems at least also intended for apiDeprecationErrors, but perhaps only intended?

Reply:
Good eye! I've noted this in the description for --apiDeprecationErrors as well.